test/expected/umd

```diff
@@ -4,6 +4,6 @@
 	factory(global.name)
 }(this, function (name) { 'use strict';
 
-	name['default']+'';
+	name+'';
```

Comment:
and now I saw some order of error in the tests: esperanto is not dealing with the UMD name. It is a necessary option but it's not being processed to the given value.